### PR TITLE
[ch38423]LMSM-Alert-email-notification

### PR DESCRIPTION
**Pull request by smihalache06**

### CODE REVIEW

File: src/etools/applications/last_mile/tasks.py [hunk 1]
```diff
@@ -35,15 +35,16 @@ def notify_upload_waybill(tenant_name, destination_pk, waybill_pk, waybill_url):
...
+def notify_wastage_transfer(tenant_name, transfer_pk, proof_file_pk, based_root, action='wastage_checkout'):
     action_map = {
         'wastage_checkout': 'checked-out as wastage',
         'short_checkin': 'checked-in as short',
         'surplus_checkin': 'checked-in as surplus'
     }
     with schema_context(tenant_name):
         transfer = models.Transfer.objects.get(pk=transfer_pk)
-
+        attachment = Attachment.objects.filter(pk=proof_file_pk).first()
```
Comment (emaciupe) [inline src/etools/applications/last_mile/tasks.py:46]: @smihalache06 please build the proof url and send it as param to the notify task

File: src/etools/applications/last_mile/tasks.py [hunk 2]
```diff
@@ -55,5 +56,26 @@ def notify_wastage_transfer(tenant_name, transfer_pk, action='wastage_checkout')
...
+                    realms__group__name='LMSM Alert Receipt') \
+            .values_list('email', flat=True) \
+            .distinct()
+
+        send_notification(
+            recipients=list(recipients),
+            from_address=settings.DEFAULT_FROM_EMAIL,
+            subject='Acknowledged by IP',
+            html_content_filename='emails/first_checkin.html',
+            context={'transfer': transfer, "waybill_urls": waybill_urls}
```
Comment (emaciupe) [inline src/etools/applications/last_mile/tasks.py:80]: @smihalache06 pls use proof file instead waybill as it is not currently used in FE

**Review by emaciupe — CHANGES_REQUESTED** [review]

File: src/etools/applications/last_mile/tasks.py (hunk 1 above)
Comment (smihalache06) [reply src/etools/applications/last_mile/tasks.py]: Solved!

File: src/etools/applications/last_mile/tasks.py (hunk 2 above)
Comment (smihalache06) [reply src/etools/applications/last_mile/tasks.py]: Solved!

**Pull request merged**